From 130463160b4923069eb98da49edaf7d93180f4f8 Mon Sep 17 00:00:00 2001
From: Antti Kokko <antti.kokko@qt.io>
Date: Tue, 27 Oct 2020 10:02:12 +0200
Subject: [PATCH 2/2] Add changes file for Qt 5.15.2
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

- d759a753779f3ae4b5ae2cb6b16401b4663618ad Add changes file for Qt 5.15.1
+ 1a25abd33bf65bae6434c7a7fdd747929ffe2d36 Bump version
+ c082168fba5deb930a01502c54bf2080af53b544 Add changes file for Qt 5.12.10
+ 07f28ce1eab983a49e661855ee11ffbd624f0083 Prevent zoom-out beyond infinite linear max double values

Change-Id: I2dabc0e6ce8f91aac3933b88d2e5b815041dc9b8
Reviewed-by: Tomi Korpipää <tomi.korpipaa@qt.io>
Reviewed-by: Miikka Heikkinen <miikka.heikkinen@qt.io>
(cherry picked from commit a67f812548b008e3eedcd2bb9313828a195fd23b)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 dist/changes-5.15.2 | 37 +++++++++++++++++++++++++++++++++++++
 1 file changed, 37 insertions(+)
 create mode 100644 dist/changes-5.15.2

diff --git a/dist/changes-5.15.2 b/dist/changes-5.15.2
new file mode 100644
index 00000000..564c6cf9
--- /dev/null
+++ b/dist/changes-5.15.2
@@ -0,0 +1,37 @@
+Qt 5.15.2 is a bug-fix release. It maintains both forward and backward
+compatibility (source and binary) with Qt 5.15.1.
+
+For more details, refer to the online documentation included in this
+distribution. The documentation is also available online:
+
+  https://doc.qt.io/qt-5.15/index.html
+
+The Qt version 5.15 series is binary compatible with the 5.14.x series.
+Applications compiled for 5.14 will continue to run with 5.15.
+
+Some of the changes listed in this file include issue tracking numbers
+corresponding to tasks in the Qt Bug Tracker:
+
+  https://bugreports.qt.io/
+
+Each of these identifiers can be entered in the bug tracker to obtain more
+information about a particular change.
+
+****************************************************************************
+*                   Important Behavior Changes                             *
+****************************************************************************
+
+****************************************************************************
+*                          Library                                         *
+****************************************************************************
+
+ - [QTBUG-75500] Prevent zoom-out beyond infinite linear max double values
+   When zooming out domains that include log axes, prevent zoom-out when
+   the new zoom level would result in a max value that is above the upper
+   range of double (represented as inf). Additionally, prevent zoom-out
+   when the log value exceeds the number of pixels in the log axis
+   dimension. Major gridlines are created at each integral log value, and
+   it does not make sense to zoom out beyond such point.
+   The bug was: Stack overflow on high zoom out of QChart with a
+   logarithmic axis
+
-- 
2.31.1

